internal/kong-config.yaml

```
@@ -0,0 +1,53 @@
+---
+_format_version: "1.1"
+services:
+  - name: kibana
+    url: https://identity-idva-monitoring-kibana-${ENVIRONMENT_NAME}.apps.internal:61443
+    routes:
+      - name: kibana-route
+        hosts:
+          - identity-idva-monitoring-kibana-${ENVIRONMENT_NAME}.apps.internal
```

For each of these, I'd rather we begin using the `paths` option in the `routes` configuration instead of matching by hostname.
```suggestion
    routes:
      - name: kibana-route
        paths:
          - /kibana
```

---

For single-element yaml lists, it may be cleaner to just inline it as well `paths: ['/kibana']`. But either way is fine with me.

---

we would need to configure proxy paths/ change the extenral urls on the services here. otherwise redirects/link break.

---

replaced hosts with paths. will need to update the configuration on the services